client/src/services/API/displayImages.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+
+/**
+ * @function displayImages - select the images from the upload table base on the user ID and the selected month
+ * @param {*} token -token provide by JWT
+ * @param {*} userId - from account table
+ * @param {*} month - select by the user
+ * @returns {array} - an array of objects
+ */
+
+
+
+
+
+export default  async function displayImages (token,userId,month){
+ 
+    const newLocal = "/upload/images"
+    const response = await axios.post(newLocal,  {
```

Comment:
Since we're selecting images from the API, technically we'd expect this call to be a GET request. Not blocking I'd say though, but maybe we can discuss it a bit later :)

Reply:
I am not sure if I am right, but I need to pass a body request to check the parameters with the database. That's why I used POST. Let's chat later, I am curious to know if it is possible with GET